src/Bloc/BlElement.class.st

```diff
@@ -1854,7 +1896,7 @@ BlElement >> hasChildren [
 	false otherwise"
 	<return: #Boolean>
 	
-	^ self children isEmpty not
+	^ self childrenCount > 0
```

Comment:
This is the only small change where I have a question. Is this based on some profiling or bench? to me it sounds faster asking `isEmpty` than counting

Reply:
Thanks @tinchodias, you have right, at the beginning I rewrote the method to remove the not necessary negation, but this is better with notEmpty

Reply:
Okay, great. I didn't check but sometimes this kind of methods are executed very often, so a small change impacts.